java/pom.xml

```diff
@@ -105,7 +105,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
-            <version>1.7.9</version>
+            <version>1.7.30</version>
```

Comment:
Nit: we should make this a property as was done for `junit.version` to make keeping the various slf4j artifacts needed easier to keep in sync.